docs/pages/management/guides/teleport-operator.mdx

```diff
@@ -248,8 +248,8 @@ $ kubectl logs "$AUTH_POD" -c operator
 
 ## Next Steps
```

Comment:
```suggestion
## Next steps
```
While we're at it, we can also make this header sentence case to match the style guide